From khali@linux-fr.org Sun Aug 13 14:36:31 2006
Date: Sun, 13 Aug 2006 23:36:27 +0200
From: Domen Puncer <domen.puncer@ultra.si>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 05/12] i2c-au1550: Add SMBus functionality flag
Message-Id: <20060813233627.024251a9.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-au1550-add-smbus-functionality-flag.patch

From: Domen Puncer <domen.puncer@ultra.si>

i2c-au1550: Add SMBus functionality flag

Add SMBus functionality flag, so we can use eeprom and similar
drivers.

Signed-off-by: Domen Puncer <domen.puncer@ultra.si>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-au1550.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-au1550.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-au1550.c
@@ -285,7 +285,7 @@ au1550_xfer(struct i2c_adapter *i2c_adap
 static u32
 au1550_func(struct i2c_adapter *adap)
 {
-	return I2C_FUNC_I2C;
+	return I2C_FUNC_I2C | I2C_FUNC_SMBUS_EMUL;
 }
 
 static struct i2c_algorithm au1550_algo = {
